### ISSUE TITLE
Unsigned values for IDL?

### ISSUE THREAD

**Issue by dtapuska** [w3c/reporting#105]
Any idea why lineNumber and columnNumber are signed longs?

I think the probably really should be: (for DeprecationReportBody and InterventionReportBody)

```
  readonly attribute unsigned long? lineNumber;
  readonly attribute unsigned long? columnNumber;
```

**Comment by paulmeyer90** [comment]
Good catch!

**Comment by paulmeyer90** [comment]
Pull request to fix: #112
